verificarlo.in.in

```diff
@@ -199,7 +188,10 @@ if __name__ == "__main__":
     parser.add_argument('-static', '--static', action='store_true', help='produce a static binary')
     parser.add_argument('--verbose', action='store_true', help='verbose output')
     parser.add_argument('--inst-fcmp', action='store_true', help='instrument floating point comparisons')
+    parser.add_argument('--fortran-linker', action='store_true', help='use fortran linker')
```

Comment:
Not sure I like `--fortran-linker`. Is there a way to have a language agnostic linking as before ?

Reply:
See the previous answer.